Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
 noDirectUsers=Keine direkten Benutzer
 permissionsDisable=Berechtigung abschalten?
 eventTypes.PERMISSION_TOKEN.name=Berechtigungstoken
+eventTypes.UPDATE_TOTP.name=Update totp
```
@robson90 - could you please create a PR for the English text that it is `Update TOTP`, as TOTP is an acronym for a time-based one-time password. In parallel, I suggest to name this `Aktualisierung TOTP`.

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
 permissionsDisable=Berechtigung abschalten?
 eventTypes.PERMISSION_TOKEN.name=Berechtigungstoken
+eventTypes.UPDATE_TOTP.name=Update totp
+subjectNameId=Betreff NameID
```
IMHO this is a technical term in SAML and therefore shouldn't be translated.

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
+artifactBindingUrl=Artifact Binding URL
+eventTypes.LOGIN.name=Anmelden
+clearFileExplain=Sind Sie sicher, dass Sie diese Datei leeren möchten?
+eventTypes.LOGOUT_ERROR.name=Logout Fehler
```
Similar to the above, I suggest for example `Fehler beim Anmelden` or `Anmeldefehler`. Please review the other `eventTypes.*.name` entries as well.

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
+flowUsedBy=Verwendung von diesem Flow
+client-updater-trusted-hosts.label=Vertrauenswürdige Hosts
+updateExecutorSuccess=Ausführung erfolgreich aktualisiert
+assertionLifespan=Behauptung Lebensdauer
```
IMHO an `Assertion` is again a SAML specific term which shouldn't be translated. Maybe `Assertion Gültigkeitsdauer`?

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
+eventTypes.UPDATE_CONSENT.name=Aktualisierung der Einwilligung
+forceArtifactBinding=Artifact Binding erzwingen
+eventTypes.REFRESH_TOKEN_ERROR.description=Refresh Token Fehler
+eventTypes.IMPERSONATE.name=Impersonieren
```
Not sure if there is a good German word for this. What about `Sich ausgeben als`? The context where this is used: An admin can impersonate a user and then act as this user for example to verify a problem the user is reporting.

ahus1 reviewed (changes requested):
Hi @robson90 - thank you for these translations. IMHO you stumbled across some terms that are specific to SAML that maybe should not be translated. See below for some comments. Thanks!

robson90 reviewed (commented):
I checked some of the `eventTypes.*` and stumbled upon these words, that are very hard to get translated right. Do you have any suggestions ?

`Introspection`
`Impersonate error`

I did also Update all those Strings to be the same. Please re-review:
- Client update
- Delete Account
- Logout
- Execute actions
- Update password

weblate committed:
Updated translation for German

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Robin Meese <39960884+robson90@users.noreply.github.com>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Robin Meese <39960884+robson90@users.noreply.github.com>

weblate committed:
Translated using Weblate (Dutch)

Translation: Keycloak/Account UI
Translate-URL: https://hosted.weblate.org/projects/keycloak/account-ui/nl/

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Jon Koops <jonkoops@gmail.com>
Signed-off-by: Jon Koops <jonkoops@gmail.com>

ahus1 commented:
@robson90 - this one took a bit longer to review than I hoped as I found out that those events are really hard to translate. @jonkoops and me were also looking into Weblate configurations to figure out if we can split different languages into to different pull requests as this would allow us to onboard additional languages, and have a review of single languages in each PR. 

I change my mind about the translation of "impersonate" ... I now translated it as "Identität annehmen". Let me know what you think of this.
